src-backend/api/apps.py

```diff
@@ -0,0 +1,20 @@
+from django.apps import AppConfig
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from models import Procedure, Page, Element
+
+
+class ApiConfig(AppConfig):
+    name = 'api'
+
+    def ready(self):
+        # Grant model permissions to the appropriate user groups
+        normal_users_group, created = Group.objects.get_or_create(name='normal_users')
+        models = [Procedure, Page, Element]
+
+        for m in models:
+            content_type = ContentType.objects.get_for_model(m)
+            permissions = Permission.objects.filter(content_type=content_type)
+
+            for p in permissions:
```

Comment:
it's a list so I could just say `normal_users_group.permissions = permissions` but this would prevent other apps from adding additional permissions to this user group